datafusion/expr/src/expr.rs

```diff
@@ -64,6 +64,15 @@ use sqlparser::ast::{
 ///
 /// [`ExprFunctionExt`]: crate::expr_fn::ExprFunctionExt
 ///
+/// # Printing Expressions
```

Comment:
I added this section and the one below to try and explain the uses of the different explain variants.